integrations/tensorflow/iree_tf_compiler/TFL/LowerGlobalTensors.cpp

```diff
@@ -112,85 +110,72 @@ class LowerGlobalTensorsPass
         loc = std::get<1>(*locIt);
       }
 
-      std::string flowSymName = "__iree_flow_" + name.str();
-
       // TODO(suderman): Determine the global type based on all store
       // operations.
-      auto global = builder.create<iree_compiler::IREE::Input::GlobalOp>(
-          loc, flowSymName, /*is_mutable=*/true, attribute.getType(),
-          attribute);
+      std::string symName = "__global_" + name.str();
```

Comment:
Is the prefix needed? I think the symbols here only has to be unique per GlobalOp so may not be needed (but we can check in follow up too)

Reply:
Removed the prefix. It should be safe.